RecoEgamma/EgammaPhotonProducers/src/TrackProducerWithSCAssociation.cc

```diff
@@ -1,21 +1,64 @@
-#include "RecoEgamma/EgammaPhotonProducers/interface/TrackProducerWithSCAssociation.h"
-// system include files
-#include <memory>
-// user include files
-#include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/MakerMacros.h"
+/** \class  TrackProducerWithSCAssociation
+ **  
+ **
+ **  \author Nancy Marinelli, U. of Notre Dame, US
+ **   Modified version of TrackProducer by Giuseppe Cerati
+ **   to have super cluster - conversion track association
+ ** 
+ ***/
+
```

Comment:
Missing
`#include "DataFormats/TrackReco/interface/TrackFwd.h"`
(and perhaps Track.h as well)